Dockerfile.xpu

```diff
@@ -1,4 +1,4 @@
-FROM intel/oneapi-basekit:2024.1.0-devel-ubuntu20.04
+FROM intel/oneapi-basekit:2024.1.0-devel-ubuntu22.04 AS vllm-base
```

Comment:
If we could do without full oneapi it would be nice to utilize `intel/intel-optimized-pytorch:2.1.40-xpu-pip-base` instead and reduce the image size by 12GB.